### PR TITLE
Update pipfile and .aicoe-ci for Thoth image build. Include simple donkeycar 

### PR DESCRIPTION
**Pull request by MichaelClifford**
## Related Issues and Dependencies
Related to  #25

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

This PR includes the following

1. An initial Pipfile and Pipefile.lock for the base donkeycar requirements.  
2. Update to .aicoe-ci.yaml to build jupyter lab base images on new tag release.
3. Add a `default_car` directory representing the minimal skeleton for a donkeycar.

### CODE REVIEW

**Comment by MichaelClifford** [comment]
@sub-mod @TreeinRandomForest

**Comment by MichaelClifford** [comment]
Precommit is failing due to the formatting of the default donkey car code. Since the donkeycar project will be integral to this project, is it recommended that we reformat all donkeycar code to meet our pre-commit requirements, or ignore some checks for these files?

**Comment by MichaelClifford** [comment]
@harshad16 @goern shall we approve this PR? Or would you prefer we remove the donkycar skeleton code causing the pre-commit issues for now?

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+donkeycar = {editable = true, ref = "81c2ea397695790c370669ce57988c403f7cf7ea", git = "https://github.com/autorope/donkeycar"}
```
Comment (goern) [inline Pipfile:7]: I dont know if 'editable' is really meaningful if run from a container image. @pacospace haven't we seen problems arise from this?

**Comment by goern** [comment]
> Precommit is failing due to the formatting of the default donkey car code. Since the donkeycar project will be integral to this project, is it recommended that we reformat all donkeycar code to meet our pre-commit requirements, or ignore some checks for these files?

I'm pretty sure you can exclude file or dir or both from pre-commit checks ;)

File: Pipfile (hunk 1 above)
Comment (pacospace) [reply Pipfile:7]: yes in https://github.com/aicoe-aiops/project-template/issues/50 and to remove that donkeycar needs to be released on some indexes.

Comment (goern) [reply Pipfile:7]: @pacospace @MichaelClifford shall we create a fork of it and release it as donkeycar-redhat to pypi?! @fridex wdyt?

Comment (MichaelClifford) [reply Pipfile:7]: We need this package for the demo. If it must be released on an index, then yes, let's create a fork and add it to pypi :+1:

**Comment by MichaelClifford** [comment]
> I'm pretty sure you can exclude file or dir or both from pre-commit checks ;) 

Ok, then I'll disable pre-commit checks for the donkeycar directory. :smiley:

File: Pipfile (hunk 1 above)
Comment (goern) [reply Pipfile:7]: see https://github.com/AICoE/donkeycar and https://github.com/AICoE/common/commit/813e4e836f2dd587b0bdf435f9567966b5e7b95c

next: someone needs to setup aicoe-ci to deliver the wheel to pypi as `donkeycar-redhat`

Comment (MichaelClifford) [reply Pipfile:7]: @goern who can setup aicoe-ci to deliver this to pypi?

Comment (codificat) [reply Pipfile:7]: I'll look at setting up aicoe-ci pipelines for the donkeycar repo. For now, AICoE/common#42 to get things started

Comment (MichaelClifford) [reply Pipfile:7]: Thanks @codificat! 
Any ETA on this?

Comment (MichaelClifford) [reply Pipfile:7]: @pacospace @goern I was able to build this successfully using meteor, so the current pipfile seems to work ok. So, my question is, do we need to hold up this PR until donkey-redhat is installable via pipy, or can that come in a later PR?

Comment (codificat) [reply Pipfile:7]: An initial version of the package is up [in PyPI](https://pypi.org/project/aicoe-donkeycar/) now if you want to try and use it. The package name is: `aicoe-donkeycar`

There's also a PR to add aicoe-ci configuration for future updates: https://github.com/AICoE/donkeycar/pull/1

Comment (MichaelClifford) [reply Pipfile]: This issue has been addressed with an updated commit. Thanks for helping me out @codificat 

@goern @pacospace

**Comment by MichaelClifford** [comment]
@goern @harshad16 @codificat @harshad16 

All comments up to now should have been addressed. Could I get another review from you all? thanks!

**Pull request merged**